<launch>
  <!-- DO NOT USE ROBOT DESCRIPTION IF NOT NEEDED  <param name="robot_description" command="$(find xacro)/xacro.py '$(find sr_description)/robots/arm_and_hand_motor.urdf.xacro'" /> -->
  <param name="sa_description"
  command="$(find xacro)/xacro.py '$(find sr_description)/robots/sr_arm_motor.urdf.xacro'" />
  <!--
           shadowarm
                        -->
  <node pkg="sr_hand" name="shadowarm" type="shadowarm_can">
    <param name="shadowhand_prefix" type="string"
    value="sr_arm/" />
    <param name="controller_suffix" type="string"
    value="position_controller" />
    <remap from="joint_states" to="/sa/joint_states" />
    <param name="publish_frequency_diagnostics" type="double"
    value="1.0" />
    <param name="publish_frequency" type="double" value="100.0" />
  </node>
</launch>
